defaults/main.yml

```diff
@@ -47,6 +47,7 @@ postgres_hbas:
 ##########
 # postgresql.conf
 ##########
+postgres_unix_socket_directories: /run/postgres
```

Comment:
Doesn't changing require a restart?

Reply:
Also, remove this from the vars if you put it to the defaults, otherwise it's ignored